src/transformations/transformations.ts

```diff
@@ -330,3 +338,16 @@ export const areAnyTransformationsOverlapping = (transformations: TextTransforma
 
   return false;
 };
+
+export const areAllSameTransformation = (transformations: Transformation[]): Boolean => {
+  const firstTransformation = transformations[0];
+
+  return transformations.every(t => {
+    for (const [key, value] of Object.entries(firstTransformation)) {
```

Comment:
I think this for loop can also be written with `every`

Reply:
Thanks, I'll get this fixed